source/includes/_school-api-guide.md

```diff
@@ -2,4 +2,417 @@
 
 Detailed explanation and examples to using Edvisor Api as a School
 
-## TO DO
+## Web Quotes Engine
+
+Blah blah blah
```

Comment:
is this part going to be updated in another task soon?

Reply:
Awooooops thanks Mila!